libs/langgraph/src/pregel/loop.ts

```diff
@@ -162,13 +169,17 @@ export class PregelLoop {
     this.streamKeys = params.streamKeys;
     this.nodes = params.nodes;
     this.skipDoneTasks = this.config.configurable?.checkpoint_id === undefined;
+    this.store = params.store;
   }
 
   static async initialize(params: PregelLoopInitializeParams) {
+    const configForManaged = patchConfigurable(params.config, {
+      [CONFIG_KEY_STORE]: params.store,
+    });
```

Comment:
For reviewer: This is the main implementation of this PR (besides the store classes, however those have tests).

This is essentially supposed to port the `__enter__` method of `(Sync|Async)PregelLoop`and some code from `ChannelsManager` (setting the config).